tests/testthat/test-shinytest2-landing_popup.R

```diff
@@ -0,0 +1,184 @@
+testthat::test_that("e2e: teal app with landing_popup_module initializes with no errors", {
+  app <- TealAppDriver$new(
+    data = simple_teal_data(),
+    modules = modules(
+      landing_popup_module(
+        title = "Welcome",
+        content = tags$b("A welcome message!", style = "color: red;")
+      ),
+      example_module()
+    )
+  )
+  app$view()
+
+  app$wait_for_idle(timeout = default_idle_timeout)
+  app$expect_no_shiny_error()
+  app$stop()
+})
+
+test_that("e2e: app with default landing_popup_module creates modal containing a button", {
+  app <- TealAppDriver$new(
+    data = simple_teal_data(),
+    modules = modules(
+      landing_popup_module(),
+      example_module()
+    )
+  )
+  app$wait_for_idle(timeout = default_idle_timeout)
+
+  expect_equal(
+    app$get_html(".btn-default") %>%
+      grep("Accept", x = ., value = TRUE) %>%
+      rvest::read_html() %>%
+      rvest::html_text(),
+    "Accept"
+  )
```

Comment:
Alternative suggestion (only buttons that have "Accept", exist inside popup):

```suggestion
  testthat::expect_setequal(
    app$get_text("#shiny-modal-wrapper .btn-default"),
    "Accept"
  )
```

Or make sure there exists only 1

```suggestion
  testthat::expect_equal(
    app$get_text("#shiny-modal-wrapper .btn-default"),
    "Accept"
  )
```

Reply:
Looks like ` app$get_text` is a gamechanger and a huge improvement in comparison to manual parsing of the text.

@averissimo your both suggestions are the same?

Reply:
Ah, there is `testthat::expect_setequal(` and `testthat::expect_equal(`

Reply:
Yup, one would technically accept multiple buttons as long as all have "Accept"